Aztec/Classes/TextKit/ParagraphStyle.swift

```diff
@@ -183,6 +183,39 @@ open class ParagraphStyle: NSMutableParagraphStyle, CustomReflectable {
         }
     }
 
+    /// The amount of indent for the blockquote of the paragraph if any.
+    ///
+    public var blockquoteIndent: CGFloat {
+        let blockquoteIndex = self.properties.filter({ (property) -> Bool in
+            return property is Blockquote || property is TextList
+        }).index(where: { (property) -> Bool in
+            return property is Blockquote
+        })
+
+        guard let depth = blockquoteIndex else {
+            return 0
+        }
+
+        return CGFloat(depth) * Metrics.listTextIndentation
+    }
+
+    /// The amount of indent for the list of the paragraph if any.
+    ///
+    public var listIndent: CGFloat {
+        let listAndBlockquotes = self.properties.filter({ (property) -> Bool in
+            return property is Blockquote || property is TextList
+        })
```

Comment:
Few notes!

- No need for `self`
- No need to wrap up the last closure within braces
- No need to define the closure's return type

```
let listAndBlockquotes = properties.filter { property in
    return property is Blockquote || property is TextList
}
```